src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -130,6 +130,26 @@ def set(self, svc_id, device_class):  # pragma: no cover
                     'ids': [svc_id]
                 })
 
+    @RESTController.Resource('PUT')
+    def flags(self, svc_id, flags):
```

Comment:
Please ensure you add the proper syntax, typing and documentation here (you may check #36347 and #35785 as a reference).